sklearn/inspection/_plot/partial_dependence.py

```diff
@@ -1112,6 +1171,34 @@ def _plot_two_way_partial_dependence(
                 ax.set_xlabel(self.feature_names[feature_idx[0]])
             ax.set_ylabel(self.feature_names[feature_idx[1]])
 
+    def add_individual_line_color_info(self, color_dict):
+        last_ax_idx_not_none = np.argwhere(self.axes_.flatten() != None)[-1][
+            0
+        ]  # is not None, doesn't work here
+        if color_dict.get("type", None) is None:
+            return None
+        if color_dict["type"] == "categorical":
```

Comment:
statt extra die typen mit zu speichern könntest du auch sowas machen wie `if "categorical_mapping" in color_dict` oder `if "cmap" in color_dict`